.github/workflows/crucible-ci.yaml

```diff
@@ -13,25 +13,24 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Show Environment
-      run: |
-        whoami
-        echo
-        pwd
-        echo
-        ls -la
-
-    - name: Git Clone crucible-ci
-      run: |
-        pushd ~/
-        git clone https://github.com/perftool-incubator/crucible-ci.git
-
-    - name: Setup CI Environment
-      run: |
-        sudo ~/crucible-ci/setup-ci-environment --ci-target uperf --ci-target-dir $(pwd)
-
-    - name: Run CI
-      run: |
-        sudo ~/crucible-ci/run-ci --scenarios uperf
+    - name: Checkout bench-uperf
+      uses: actions/checkout@v2
+      with:
+        path: bench-uperf
```

Comment:
question: is this the same ?
```
- uses: actions/checkout@v2
```

Reply:
the same as what?

Reply:
we are using path to avoid conflicts between the two repos we are checking out